### DISCUSSION TITLE
How to filter using embed component

### DISCUSSION THREAD

**Discussion by ShivrajJoshi1996** [Budibase/budibase#9171] [Help]
Hi @melohagan ,

I need your help on this.

I am trying to filter table data using radio button and I don't want to select it manually. I didn't see any option in budibase according to my requirement so I tried to create one custom radio button using embed component.

I am able to achieve what I was trying but now issue is I can't use embed in filtering.

Can you please help me on this how can I use embed in filtering or if there is any other approach.

Thanks.

**Comment by melohagan** [comment]
Hey @ShivrajJoshi1996 

I don't believe there is a way to use an Embed component to filter a Data provider.

What is your use-case?
